0000-template.md

```diff
@@ -1,102 +1,118 @@
 ---
-feature name: (fill me in with a unique identifier, my-awesome-feature)
-start date: (fill me in with today's date, YYYY-MM-DD)
-rfc pr: (leave this empty)
-related issue: (tracking issue number)
+rfc pr: https://github.com/aws/aws-cdk-rfcs/pull/xxx <-- fill this after you've already created the PR
+related issue: https://github.com/aws/aws-cdk-rfcs/issues/yyy
 ---
 
-# Summary
+# [TITLE]
 
-> Brief description of the feature.
+> One sentence: brief description of the feature from a user perspective.
 
-# README
+## Working Backwards
 
-> Work backwards from the README: pretend that this feature is already
-> implemented and write the README section. It will help you think about your
-> proposal from a user experience point of view. If a README format does not
-> make sense for your proposal, write a "Press Release" which announces your new
-> feature instead.
+> This section should contain one or more "artifacts from the future", as if the
+> feature was already released and we are publishing its CHANGELOG, README
+> and optionally a PRESS RELEASE.
+>
+> This is the most important section of your RFC. It's a powerful thought
+> exercise which will challenge you to truly think about this feature from a
+> user's point of view.
 
-# Motivation
+### CHANGELOG
 
-> Why are we doing this? What use cases does it support? What is the expected
-> outcome?
+> Write the changelog entry for this feature in conventional form (e.g.
+> `feat(eks): cluster tags`).
+>
+> If this change includes a breaking change, include a `BREAKING CHANGE` clause
+> with information on how to migrate. If migration is complicated, refer to a
+> fictional GitHub issue and add its contents here.
 
-# Design Summary
+### README
 
-> Summarize the approach of the feature design in a couple of sentences. Call out
-> any known patterns or best practices the design is based around.
+> If this is a new feature, write the README section which describes this new
+> feature. It should describe the feature and walk users through usage examples
+> and description of the various options and behavior.
 
-# Detailed Design
+### PRESS RELEASE
 
-> This is the bulk of the RFC. Explain the design in enough detail for somebody
-> familiar with CDK to understand, and for somebody familiar with the
-> implementation to implement. This should get into specifics and corner-cases,
-> and include examples of how the feature is used. Any new terminology should be
-> defined here.
->
-> Include any diagrams and/or visualizations that help to demonstrate the design.
-> Here are some tools that we often use:
->
-> - [Graphviz](http://graphviz.it/#/gallery/structs.gv)
-> - [PlantText](https://www.planttext.com)
+> If this is a major feature (~6 months of work), write the PRESS RELEASE which
```

Comment:
This is covered in Working Backwards above. We should merge these paragraphs.

Reply:
This is the working backwards section. Not sure I follow...